packages/firestore/lite/index.d.ts

```diff
@@ -45,6 +45,7 @@ export function setLogLevel(logLevel: LogLevel): void;
 
 export interface FirestoreDataConverter<T> {
   toFirestore(modelObject: T): DocumentData;
+  toFirestore(modelObject: Partial<T>, options: SetOptions): DocumentData;
```

Comment:
Note that this change also need to be made in exp/index.d.ts

Reply:
done for converter and `SetOptions`.